marlowe-dashboard-client/src/ContractHome/View.purs

```diff
@@ -108,18 +108,18 @@ contractCard currentSlot contractState =
     div
       -- NOTE: The overflow hidden helps fix a visual bug in which the background color eats away the border-radius
       [ classNames
-          [ "cursor-pointer", "shadow", "bg-white", "rounded", "overflow-hidden" ]
+          [ "cursor-pointer", "shadow-sm", "hover:shadow", "active:shadow-lg", "bg-white", "rounded", "overflow-hidden" ]
       , onClick_ $ OpenContract contractInstanceId
       ]
-      [ div [ classNames [ "flex", "px-4", "pt-4" ] ]
-          [ span [ classNames [ "text-xl", "font-semibold" ] ] [ text contractAcronym ]
-          , span [ classNames [ "flex-grow", "text-xs" ] ] [ text contractType ]
-          , icon_ ArrowRight
+      [ div [ classNames [ "flex", "px-4", "pt-4", "items-center" ] ]
+          [ span [ classNames [ "text-2xl", "leading-none", "font-semibold" ] ] [ text contractAcronym ]
+          , span [ classNames [ "flex-grow", "ml-2", "self-start", "text-xs", "uppercase" ] ] [ text contractType ]
+          , icon ArrowRight [ "text-28px" ]
```

Comment:
There's a `contractTitle` view function in `Template.View` which needs fixing as well. And ideally we should factor that out somewhere so we're using the same view function in both places.

Reply:
I've made the corresponding changes in Template.View, and I've put a comment about possible duplication, but I'm not sure about the factoring out just yet as they only share the two letters and the upper part. The arrow and the long title is missing from the template.